checkov/kubernetes/checks/resource/k8s/KubeletClientCa.py

```diff
@@ -13,16 +13,14 @@ def __init__(self) -> None:
 
     def scan_container_conf(self, metadata: Dict[str, Any], conf: Dict[str, Any]) -> CheckResult:
         self.evaluated_container_keys = ["command"]
-        if conf.get("command"):
+        if conf.get("command") is not None:
```

Comment:
```suggestion
        if conf.get("command"):
```
no need to especially check for `None` the previous condition checks for `is not None`  and also that it is not empty